<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorDesign</class>
 <widget class="QWidget" name="SensorDesign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>传感器设计</string>
  </property>
  <widget class="QLabel" name="Tietlel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>181</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>181</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#00aaff;&quot;&gt;传感器设计&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="bg_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>601</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>901</width>
     <height>601</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>901</width>
     <height>601</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:700; color:#00aaff;&quot;&gt;传感器组设置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>881</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    border: 2px solid white;
    border-radius: 5px;
    margin-top: 1.5ex;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 3px;
    color: white;
    font-weight: bold;
}</string>
   </property>
   <property name="title">
    <string>传感器组设置</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>21</y>
      <width>876</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QTableView" name="tableView_1">
       <property name="minimumSize">
        <size>
         <width>874</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>874</width>
         <height>191</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableView {
    background-color: #ffffff;
    gridline-color: #d0d0d0;
    border: 1px solid #999999;
    selection-background-color: #66aaff;
    selection-color: white;
    alternate-background-color: #f0f0f5;
}

QTableView::item {
    padding: 5px;
    font-size: 12px;
    color: #333333;
}

QTableView::item:selected {
    background-color: #66aaff;
    color: white;
}

QTableView::horizontalHeader {
    background-color: #e0e0e0;
    border-bottom: 1px solid #999999;
}

QTableView::horizontalHeader::section {
    padding: 5px;
    font-weight: bold;
    border-right: 1px solid #d0d0d0;
}

QTableView::verticalHeader {
    background-color: #e0e0e0;
    border-right: 1px solid #999999;
}

QTableView::verticalHeader::section {
    padding: 5px;
    font-weight: bold;
    border-bottom: 1px solid #d0d0d0;
}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignRight">
      <widget class="QPushButton" name="SaveButton1">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>184</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 15px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 9em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}

QPushButton:hover
{
		color:#f3f3f3;
	background-color:rgb(102,178,255); /*改变背景色*/
	border-style:inset;/*改变边框风格*/
	padding-left:6px;
	padding-top:6px;
}</string>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>881</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    border: 2px solid white;
    border-radius: 5px;
    margin-top: 1.5ex;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 3px;
    color: white;
    font-weight: bold;
}</string>
   </property>
   <property name="title">
    <string>传感器设置</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>861</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QTableView" name="tableView_2">
       <property name="styleSheet">
        <string notr="true">QTableView {
    background-color: #ffffff;
    gridline-color: #d0d0d0;
    border: 1px solid #999999;
    selection-background-color: #66aaff;
    selection-color: white;
    alternate-background-color: #f0f0f5;
}

QTableView::item {
    padding: 5px;
    font-size: 12px;
    color: #333333;
}

QTableView::item:selected {
    background-color: #66aaff;
    color: white;
}

QTableView::horizontalHeader {
    background-color: #e0e0e0;
    border-bottom: 1px solid #999999;
}

QTableView::horizontalHeader::section {
    padding: 5px;
    font-weight: bold;
    border-right: 1px solid #d0d0d0;
}

QTableView::verticalHeader {
    background-color: #e0e0e0;
    border-right: 1px solid #999999;
}

QTableView::verticalHeader::section {
    padding: 5px;
    font-weight: bold;
    border-bottom: 1px solid #d0d0d0;
}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>115</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 1px 2px 1px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 9em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}
/*QCombobox右侧按钮*/
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;/*放于右方顶部*/
    width: 50px;/*设置按钮范围宽度*/
 	/*border-radius: 15px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;*/

    border-top-right-radius: 3px;/*设置边框圆角*/
    border-bottom-right-radius: 3px;
/*padding-right: 50px;*/
}
/*QCombobox右侧按钮的箭头图标*/
QComboBox::down-arrow {
	border-image: url(:/image/down_list.png);/*自定义图片填充*/
	width: 10px;/*设置该图标的宽高*/
	height: 10px;
}


/* 下拉后，整个下拉窗体样式 */
QComboBox QAbstractItemView {
    border: 2px solid #f3f3f3;/*边框宽度、线形、颜色*/
	background-color: rgba(237, 242, 255, 1);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 1px 2px 1px 2px;  /*针对于组合框中的文本内容*/
    min-width: 9em;   /*# 组合框的最小宽度*/
}

/* 下拉后，整个下拉窗体每项的样式 */
QComboBox QAbstractItemView::item {
	border-radius: 15px;/*圆角*/
    height: 30px;   /* 项的高度（设置pComboBox-&gt;setView(new QListView());后，该项才起作用） */
	background-color: rgb(237, 242, 255);

}

/*以下部分不知为何不生效，有待调试*/
/* 下拉后，整个下拉窗体越过每项的样式 */
QComboBox QAbstractItemView::item:hover {
    color: #FFFFF0;
       /* 整个下拉窗体越过每项的背景色 */
	background-color: rgb(98, 0, 255);
}

/* 下拉后，整个下拉窗体被选择的每项的样式 */
QComboBox QAbstractItemView::item:selected {
    color: #FFFFF0;
	background-color: rgb(0, 85, 200);
}</string>
         </property>
         <item>
          <property name="text">
           <string>请选择</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>传感器组1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>传感器组2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>传感器组3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>传感器组4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>传感器组5</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DeleteButton">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 15px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 9em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}

QPushButton:hover
{
		color:#f3f3f3;
	background-color:rgb(102,178,255); /*改变背景色*/
	border-style:inset;/*改变边框风格*/
	padding-left:6px;
	padding-top:6px;
}</string>
         </property>
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AddButton">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 15px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 9em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}

QPushButton:hover
{
		color:#f3f3f3;
	background-color:rgb(102,178,255); /*改变背景色*/
	border-style:inset;/*改变边框风格*/
	padding-left:6px;
	padding-top:6px;
}</string>
         </property>
         <property name="text">
          <string>增加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SaveButton2">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>184</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 15px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 9em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}

QPushButton:hover
{
		color:#f3f3f3;
	background-color:rgb(102,178,255); /*改变背景色*/
	border-style:inset;/*改变边框风格*/
	padding-left:6px;
	padding-top:6px;
}</string>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>bg_label</zorder>
  <zorder>Tietlel</zorder>
  <zorder>label</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
